tools/manifest-templator/manifest-templator.go

```diff
@@ -224,6 +224,7 @@ func main() {
 	containerTag := flag.String("container-tag", "latest", "The operator image's container tag")
 	imagePullPolicy := flag.String("image-pull-policy", "Always", "The pull policy to use on the operator image")
 	multusImage := flag.String("multus-image", components.MultusImageDefault, "The multus image managed by CNA")
+	multusControlerImage := flag.String("multus-controller-image", components.MultusControllerImageDefault, "The multus controller image managed by CNA")
```

Comment:
Not relevant to this flag only, but are we happy to see a missing flag the hard way (like, having the process crashing because of nil pointer)?